content/en/opentelemetry/config/environment_variable_support.md

```diff
@@ -71,38 +93,155 @@ Mapped values between `OTEL_TRACES_SAMPLER` & `DD_TRACE_SAMPLE_RATE`:<br>
+
+`OTEL_EXPORTER_OTLP_LOGS_TIMEOUT`
+: **Description**: Specifies the timeout (in milliseconds) for a single outgoing OTLP logs request. Takes precedence over the general `OTEL_EXPORTER_OTLP_TIMEOUT`. <br>
+**Default**: `10000` (10s).
+
+## OpenTelemetry Metrics SDK configuration
+
+Datadog SDKs support the following OpenTelemetry Metrics SDK options.
+
+`OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE`
+: **Description**: Specifies the aggregation `temporality` to use for each instrument kind. <br>
+**Accepted values**: `Cumulative`, `Delta`, `LowMemory`. For more details, see the [OpenTelemetry specification][15]. <br>
+**Default**: `delta` <br>
+**Notes**: This default value `delta` is [Datadog's recommended configuration][16] and differs from the OpenTelemetry specification's default.
+
+`OTEL_METRIC_EXPORT_INTERVAL`
+: **Description**: Specifies the time interval (in milliseconds) between metric export attempts. <br>
+**Default**: `10000` (10s) <br>
+**Notes**: This default value is Datadog's recommended configuration and differs from the OpenTelemetry specification's default of 60000ms.
+
+`OTEL_METRIC_EXPORT_TIMEOUT`
+: **Description**: Specifies the maximum allowed time (in milliseconds) to collect and export metrics. <br>
+**Default**: `7500` (7.5s) <br>
+**Notes**: This default value is Datadog's recommended configuration and differs from the OpenTelemetry specification's default of 30000ms.
+
+## OpenTelemetry Logs SDK configuration
+
+Datadog SDKs support the following OpenTelemetry Logs SDK options.
+
+`OTEL_LOGS_EXPORTER`
+: **Description**: Specifies the logs exporter to be used. <br>
+**Accepted values**: `otlp`, `none`. A value of `none` disables the emission of OTel logs. <br>
+**Default**: `otlp`
+
+`OTEL_BLRP_MAX_QUEUE_SIZE`
+: **Description**: The maximum number of log records to hold in memory. New records are dropped when the limit is reached. <br>
+**Default**: `2048`
+
+`OTEL_BLRP_SCHEDULE_DELAY`
+: **Description**: The time interval (in milliseconds) between two consecutive export operations. <br>
+**Default**: `1000` (1s)
+
+`OTEL_BLRP_EXPORT_TIMEOUT`
+: **Description**: The maximum duration (in milliseconds) allowed for a single export before cancellation. <br>
+**Default**: `30000` (30s)
+
+`OTEL_BLRP_MAX_EXPORT_BATCH_SIZE`
+: **Description**: The maximum number of log records in a single OTLP payload. <br>
+**Default**: `512`
+
 ## Java-specific configuration
 Datadog SDKs support the following Java-specific OpenTelemetry configuration options. For more information, see the [OpenTelemetry documentation on Java agent configuration][10].
 
-
 `OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED`
-: ****Datadog convention****: `!DD_INTEGRATIONS_ENABLED` <br>
+: **Datadog convention**: `!DD_INTEGRATIONS_ENABLED` <br>
 Set to `false` to disable all instrumentation in the agent<br>
```

Comment:
```suggestion
Set to `false` to disable all instrumentation in the Agent<br>
```
noticed this was not capitalized before but just correcting it here, unless this refers to an OTEL agent ?